src/components/Search/SearchComponent.tsx

```diff
@@ -125,6 +120,34 @@ function SearchComponent({
     }
   };
 
+  const onCancel = () => {
+    if (isBaseComp) handleCancel(true);
+    if (!isBaseComp) handleCancel(false);
```

Comment:
You could simply use:
`handleCancel(isBasecomp)`

Please don't bother changing this because as I wrote above, I believe we can get rid of `isBaseCamp` altogether, we'll do that in a next PR. I just wanted to point out that these if conditions were not necessary, rather you could use a more linear code.

Reply:
True, not sure why I wrote it that. Must be tired. Will fix in next PR